client/state/lib/middleware.js

```diff
@@ -143,7 +142,6 @@ const handler = ( dispatch, action, getState ) => {
 			}, 0 );
 			return;
 
-		case SITE_DELETE_RECEIVE:
 		case JETPACK_DISCONNECT_RECEIVE:
```

Comment:
We want to remove the `JETPACK_DISCONNECT_RECEIVE` case, too, don't we?

Reply:
Of course. I guess I didn't push this change, although I had it locally.